modules/terraform/output.go

```diff
@@ -5,12 +5,21 @@ import (
 	"errors"
 	"fmt"
 	"reflect"
+	"regexp"
 	"strconv"
+	"strings"
 
 	"github.com/gruntwork-io/terratest/modules/testing"
 	"github.com/stretchr/testify/require"
 )
 
+const skipJsonLogLine = " msg="
+
+var (
+	ansiLineRegex = regexp.MustCompile(`(?m)^\x1b\[[0-9;]*m.*`)
```

Comment:
nit: can we add comments on what these regex mean in plain English?

Reply:
Updated